### PR TITLE
Fix two typos related to the Hessians.

### PR DESCRIPTION
**Pull request by drwells**
This little PR fixes two copy and paste documentation errors for second derivative code.

### CODE REVIEW

File: include/deal.II/fe/fe_values.h [hunk 1]
```diff
@@ -2026,7 +2026,7 @@ class FEValuesBase :
    *
    * @post <code>laplacians[q]</code> will contain the Laplacian of the field
    * described by fe_function at the $q$th quadrature point.
-   * <code>gradients[q][i][j]</code> represents the $(i,j)$th component of the
+   * <code>laplacians[q][i][j]</code> represents the $(i,j)$th component of the
    * matrix of second derivatives at quadrature point $q$.
```
Comment (kronbichler) [inline include/deal.II/fe/fe_values.h]: `laplacians[q][i][j]` seems wrong. The Laplacians are a scalar for a scalar element evaluation as is done here, so one should not talk about i and j here. How about saying `laplacians[q]` represents the sum of the diagonal entries of the Hessian matrix of second derivatives at quadrature point $q$?

Comment (drwells) [inline include/deal.II/fe/fe_values.h]: Yes, you are correct. I will fix this in a moment.

Comment (drwells) [inline include/deal.II/fe/fe_values.h]: come to think of it, this whole sentence should be removed

Comment (kronbichler) [inline include/deal.II/fe/fe_values.h]: I agree, but you might still add that the Laplacian computes as the trace of the Hessian, i.e., the sum of the diagonal entries in the Hessian matrix.

Comment (drwells) [inline include/deal.II/fe/fe_values.h]: I agree that we should mention that. The next `@post` in this function specifies that the trace of the Hessian is the Laplacian so we do not need to add it.

Comment (kronbichler) [inline include/deal.II/fe/fe_values.h]: I didn't look below, thanks for the clarification.

**Pull request merged**